flathub.json

```diff
@@ -0,0 +1,4 @@
+
+{
+    "only-arches": ["x86_64"]
```

Comment:
This is most likely due to a limitation in onnxruntime, you'll have to take that up with youwin to get it working on aarch64 and other architectures. If it will work natively on the M1, maybe I can get it working for aarch64 on the flatpak (with a different binary pulled)